app/services/policies/policies_list_service.rb

```diff
@@ -61,4 +56,21 @@ def to_h
       @sets.transform_values(&:to_a).as_json
     end
   end
+
+  HTTP_ERRORS = [HTTPClient::BadResponseError, HTTPClient::TimeoutError, HTTPClient::ConnectTimeoutError, HTTPClient::SendTimeoutError, HTTPClient::ReceiveTimeoutError, SocketError].freeze
+  private_constant :HTTP_ERRORS
+
+  def self.fetch_policies_from_apicast
+    return unless (apicast_registry_url = ThreeScale.config.sandbox_proxy.apicast_registry_url)
+    begin
+      response = ::JSONClient.get(apicast_registry_url)
+      response.body['policies'] if response.ok?
+    rescue *HTTP_ERRORS => error
+      Rails.logger.error("APIcast could not be found at url: '#{apicast_registry_url}'. Error: #{error}")
+      # TODO: notify UI that something went wrong
+      return nil
```

Comment:
Ok, I see that it would work 👍 :
![image](https://user-images.githubusercontent.com/11318903/56028415-564d1000-5d18-11e9-99cf-d1f326e50a8a.png)

Reply:
Ah right! I remember now I did that